### ISSUE TITLE
Use a data store instead of putting reviews in git

### ISSUE THREAD

**Issue by schradert** [ItsiW/VILF#19]
I think the main motivations for this are:
1. Keeping photos in git creates unnecessary bloat.
2. The git workflow is not ideal for non-technical people to contribute.

I think the form at #11 already addresses a much improved workflow for other people actually.
But with this workflow, it shifts work onto @ItsiW to upload people's reviews when he can into git.
If it's not a problem, then we can keep the reviews in git but have @ItsiW upload photos to the bucket.
This way, we solve the problem with the first motivation too without changing much.

If we want to move reviews out of git too, that requires using some kind of static file store on the bucket.
Maybe this can just be JSON format? We could provide a script for @ItsiW to easily migrate new submissions
to this new data store. Automated triggers might be overkill (and could cost additional money, but maybe not much,
especially if Climax is bankrolling).

**Comment by ItsiW** [comment]
Yes bucketed photos make sense. As it's a static website, how should we handle rebuilds?

I think if we want to move the reviews out of git we might as well go for a full submission tool (i.e. a UI for the reviewer to input reviews).

Lmao $$ but if we can automate as much as possible, that'd be way better
